coldfront/core/user/forms.py

```diff
@@ -78,9 +80,12 @@ def __init__(self, *args, **kwargs):
         self.middle_name = ''
         super().__init__(*args, **kwargs)
 
+    def clean(self):
```

Comment:
The `clean` method is defined by the parent class, and is run automatically whether it's defined here or not.

Given that we're not adding any custom logic to it, there should be no need to include it. We can also remove the `cleaned_data = {}` line above.